### ISSUE TITLE
Selectors containing the word "body" or "html" treated as root

### ISSUE THREAD

**Issue by GUI** [dbtedman/postcss-prefixwrap#6]
With the default configuration some selectors that contain "body" end up getting treated as the root body/html tag. So, for example, with a prefix of `.my-custom-wrap`, something like:

```css
tbody {
  border: 10px solid red;
}
```

Will get turned into:

```css
.my-custom-wrap {
  border: 10px solid red;
}
```

Whereas, this should become:

```css
.my-custom-wrap tbody {
  border: 10px solid red;
}
```

This appears to be getting tripped up by the `prefixRootTags` regex, since that's matching any selector that contains either body or html anywhere in the selector. As another example, the class name `.foobar-body` or `.foobar-html` will also gets caught by the same issue.

I was able to solve this by turning on the `prefixRootTags` option, since in my case, I didn't necessarily need this logic to extract the body or html selector styles. But I think the issue might be that the regex is too aggressive. I haven't tested this, but maybe something more like `/(^|[\s,]*)(body|html)([\s,]*|$)/` might help (that attempts to only match body or html surrounded by spaces, commas, or the beginning/end of line, but I'm not entirely sure about all the possible combinations).

Anyway, just thought I'd mention this and the potential workaround (if you don't need to extract those styles), but thanks for the plugin!

**Comment by dbtedman** [comment]
Hi @GUI, Thanks for providing this feedback. I will add additional tests to cover this issue and take a look at that regex. It looks like we need to make sure that the body tag does not have any text before it, this should avoid tbody from being mistaken for body.

**Comment by dbtedman** [comment]
Hi @GUI, this fix has been released with version 1.1.3. Thanks again for the feedback.
